### PR TITLE
[admin] Adds Spawn Human verb

### PR DESCRIPTION
**Pull request by swissloaf**
### Intent of your Pull Request

Convenience and quality of life thing. Spawns mindless humans in one click
ivanmixo bee 1505

#### Changelog

:cl:  
rscadd: Adds spawn human verb
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
amdin pwodewrcreep admin power creeps

**Comment by swissloaf** [comment]
why is turdis failing didnt even touch those lines in the code

**Comment by jcatjmeow** [comment]
debug menu
type carbon/human
click it
done

**Comment by swissloaf** [comment]
> debug menu
> type carbon/human
> click it
> done

mighty cool bro

**Comment by ynot01** [comment]
put it on right click menu for easier

**Comment by ToGWtF** [comment]
tab
spawn carbon/human
up arrow once
enter

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1233,4 +1233,16 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
-	p.throw_at(user, 10, 0.5, usr)
\ No newline at end of file
+	p.throw_at(user, 10, 0.5, usr)
+	
+/client/proc/spawnhuman()
+	set name = "Spawn human"
+	set desc = "Spawns a mindless human"
+	set category = "Fun"
+
+	if(!check_rights(R_FUN))
```
Comment (Bibby0110) [inline code/modules/admin/verbs/randomverbs.dm]: Check for R_SPAWN

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (Bibby0110) [reply code/modules/admin/verbs/randomverbs.dm]: Don’t forget to move it in the list too

**Comment by swissloaf** [comment]
> tab
> spawn carbon/human
> up arrow once
> enter

convenience, you dont have to type or shit now. you can just go to the tile you want to spawn and click click click

**Pull request closed without merging**